larq_compute_engine/tflite/benchmark/BUILD

```diff
@@ -7,12 +8,12 @@ package(
     licenses = ["notice"],  # Apache 2.0
 )
 
-cc_binary(
```

Comment:
What does `tf_cc_binary` do? Should we use it for every tflite related bazel target?

Reply:
Basically it adds a bunch of linkopts and other dependencies in order to make builds/tests work when using `framework_shared_object`, which is the reason I originally added them. We're not actually using `framework_shared_object`, but I figured that it doesn't hurt to add them. Also for some reason the benchmark binary is currently failing to build on `android_arm64` so I thought adding this might fix that (alas it did not).